simulator/Executive.ts

```diff
@@ -251,8 +254,8 @@ export class Executive implements IExecutive {
         var bInstruction = context.core.getAt(context.bPointer);
 
         bInstruction.aOperand.address = context.core.wrap(
-                aInstruction.aOperand.address + bInstruction.aOperand.address
```

Comment:
Whitespace is important! I'm not going to start on your 4 spaces heresy!

Reply:
Shift+Alt+F FTW

Reply:
Also double quotes boyee!!